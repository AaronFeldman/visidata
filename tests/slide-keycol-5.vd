sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	OrderDate		key-col		!	toggle current column as a key column
sample	Region		key-col		!	toggle current column as a key column
sample	Rep		key-col		!	toggle current column as a key column
sample	Item		key-col		!	toggle current column as a key column
sample	Item		slide-left		H	slide current column left
sample	Item		slide-left		H	slide current column left
sample	Item		slide-right		L	slide current column right
sample	Item		slide-right		L	slide current column right
sample	Item		slide-left		H	slide current column left
sample	Item		slide-left		H	slide current column left
