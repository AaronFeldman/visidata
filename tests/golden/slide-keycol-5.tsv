OrderDate	Item	Region	Rep	Units	Unit_Cost	Total
2016-01-06	Pencil	East	Jones	95	1.99	189.05
2016-01-23	Binder	Central	Kivell	50	19.99	999.50
2016-02-09	Pencil	Central	Jardine	36	4.99	179.64
2016-02-26	Pen	Central	Gill	27	19.99	539.73
2016-03-15	Pencil	West	Sorvino	56	2.99	167.44
2016-04-01	Binder	East	Jones	60	4.99	299.40
2016-04-18	Pencil	Central	Andrews	75	1.99	149.25
2016-05-05	Pencil	Central	Jardine	90	4.99	449.10
2016-05-22	Pencil	West	Thompson	32	1.99	63.68
2016-06-08	Binder	East	Jones	60	8.99	539.40
2016-06-25	Pencil	Central	Morgan	90	4.99	449.10
2016-07-12	Binder	East	Howard	29	1.99	57.71
2016-07-29	Binder	East	Parent	81	19.99	1619.19
2016-08-15	Pencil	East	Jones	35	4.99	174.65
2016-09-01	Desk	Central	Smith	2	125.00	250.00
2016-09-18	Pen Set	East	Jones	16	15.99	255.84
2016-10-05	Binder	Central	Morgan	28	8.99	251.72
2016-10-22	Pen	East	Jones	64	8.99	575.36
2016-11-08	Pen	East	Parent	15	19.99	299.85
2016-11-25	Pen Set	Central	Kivell	96	4.99	479.04
2016-12-12	Pencil	Central	Smith	67	1.29	86.43
2016-12-29	Pen Set	East	Parent	74	15.99	1183.26
2017-01-15	Binder	Central	Gill	46	8.99	413.54
2017-02-01	Binder	Central	Smith	87	15.00	1305.00
2017-02-18	Binder	East	Jones	4	4.99	19.96
2017-03-07	Binder	West	Sorvino	7	19.99	139.93
2017-03-24	Pen Set	Central	Jardine	50	4.99	249.50
2017-04-10	Pencil	Central	Andrews	66	1.99	131.34
2017-04-27	Pen	East	Howard	96	4.99	479.04
2017-05-14	Pencil	Central	Gill	53	1.29	68.37
2017-05-31	Binder	Central	Gill	80	8.99	719.20
2017-06-17	Desk	Central	Kivell	5	125.00	625.00
2017-07-04	Pen Set	East	Jones	62	4.99	309.38
2017-07-21	Pen Set	Central	Morgan	55	12.49	686.95
2017-08-07	Pen Set	Central	Kivell	42	23.95	1005.90
2017-08-24	Desk	West	Sorvino	3	275.00	825.00
2017-09-10	Pencil	Central	Gill	7	1.29	9.03
2017-09-27	Pen	West	Sorvino	76	1.99	151.24
2017-10-14	Binder	West	Thompson	57	19.99	1139.43
2017-10-31	Pencil	Central	Andrews	14	1.29	18.06
2017-11-17	Binder	Central	Jardine	11	4.99	54.89
2017-12-04	Binder	Central	Jardine	94	19.99	1879.06
2017-12-21	Binder	Central	Andrews	28	4.99	139.72
